xrviz/control.py

```diff
@@ -11,24 +11,29 @@
 
 class Control(SigSlot):
     """
-    This section allows the user to control the other subsections,
-    such as displayer, fields.
+    This section arranges all the panes in the form of tabs,
+    hence providing the controls to inteact with all the
+    panes.
 
     Parameters
     ----------
-    data: `xarray` instance: `DataSet` or `DataArray`
-           datset is used to initialize.
+    data: `xr.core.dataarray.DataWithCoords`
+        Data is required for initialization.
```

Comment:
This is obvious from the signature, and that you haven't said (optional) or given a default. Instead, we want to say what the role of this input is.